models/fivetran_platform__audit_table.sql

```diff
@@ -153,7 +153,7 @@ final as (
 
     select 
         *,
-        {{ dbt_utils.generate_surrogate_key(['schema_name','connector_id', 'destination_id', 'table_name', 'write_to_table_start']) }} as unique_table_sync_key, -- for incremental materialization 
+        {{ dbt_utils.generate_surrogate_key(['schema_name','connection_id', 'destination_id', 'table_name', 'write_to_table_start']) }} as unique_table_sync_key, -- for incremental materialization 
```

Comment:
Should we call out in the CHANGELOG this update to the surrogate key, since these values are now changing?

Reply:
I say it's covered by[ the renamed section of the changelog.](https://github.com/fivetran/dbt_fivetran_log/blob/bee80b820ed8b5c43f0da30c649b206335479279/CHANGELOG.md?plain=1#L27) I don't want to add more to lines to the changelog since it's getting a bit long.